util.py

```diff
@@ -63,18 +50,54 @@ def check(payload):
 		await channel.send("timed out")
 		raise e
 
-	moveId = RESPONSE_REACTIONS.index(str(payload.emoji))
+	reactionId = RESPONSE_REACTIONS.index(str(payload.emoji))
 	embed.clear_fields()
-	move = battlecontext.pokemon.Moves[moveId]
-	embed.add_field(
-		name=f"{move['Name']}",
-		value=
-		f"{taggify(type_to_string(move['Type']))} {move['CurrentPP']}/{move['MaxPP']}",
-		inline=False
+	return reactionId
+
+
+async def prompt_for_turn(
+	bot: commands.Bot,
+	user: discord.User,
+	battlecontext: BattleContext,
+	use_channel: Optional[discord.TextChannel] = None
+) -> Turn:
+	"""Prompt the given user for a turn.
+
+	:param bot: The discord bot
+	:param user: The discord user
+	:param battlecontext: The :class:`BattleContext` that will be given to the user.
+	:param use_channel: Override the channel that is used to communicate with the user. By default, it will use the user's :class:`discord.DMChannel` to send the prompt. Primarily used for tests.
+	:returns: The turn that the user made.
+	TODO: prompt user for what type of turn
+	"""
+
+	menu_items = []
+	menu_items.append(battlecontext.pokemon.Name)
+	menu_items.append(
+		f"{battlecontext.pokemon.CurrentHP} HP {taggify(type_to_string(battlecontext.pokemon.Type))} {taggify(status_to_string(battlecontext.pokemon.StatusEffects))}"
 	)
-	await msg.edit(content="Selected", embed=embed)
 
-	# TODO: prompt for which pokemon to target in double battles
+	for i, move in enumerate(battlecontext.pokemon.Moves):
+		menu_items.append(f"{RESPONSE_REACTIONS[i]}: {move['Name']}")
+		menu_items.append(
+			f"{taggify(type_to_string(move['Type']))} {move['CurrentPP']}/{move['MaxPP']}"
+		)
+
+	selected = await menu(bot, user, menu_items, use_channel)
+	move = battlecontext.pokemon.Moves[selected]
+
+	menu_items = []
+	menu_items.append(battlecontext.pokemon.Name)
+	menu_items.append(
+		f"{battlecontext.pokemon.CurrentHP} HP {taggify(type_to_string(battlecontext.pokemon.Type))} {taggify(status_to_string(battlecontext.pokemon.StatusEffects))}"
+	)
+	menu_items.append(f"{move['Name']}")
+	menu_items.append(
+		f"{taggify(type_to_string(move['Type']))} {move['CurrentPP']}/{move['MaxPP']}"
+	)
```

Comment:
I'm not sure why you're trying to do it like this. I don't understand why you're trying to put everything in one list. You're limiting yourself for no reason

Reply:
I put everything in one list since menu() only takes one list as an input, unless I'm wrong?

Reply:
I guess I should have said:
- has to _at minimum, in any way, shape, or form_ take a list of menu items.
- must have convenient and clear arguments to make it easy to use
- must be able to reproduce the existing `prompt_for_turn` menu exactly as it is now

You can do literally anything you need to (create new classes, add arguments, whatever) to make it **as convenient as possible** to make menus.